### PR TITLE
Fix bug uncovered during deploy

### PR DESCRIPTION
**Pull request by ericdill**
attn @tacaswell

### CODE REVIEW

File: suitcase/spec.py [hunk 1]
```diff
@@ -740,13 +739,13 @@ def to_spec_file_header(start, filepath, baseline_descriptor=None):
...
 
-_SPEC_1D_COMMAND_TEMPLATE = env.from_string("{{ plan_type }} {{ scan_motor }} {{ start }} {{ stop }} {{ num }} {{ time }}")
+_SPEC_1D_COMMAND_TEMPLATE = env.from_string("{{ plan_name }} {{ scan_motor }} {{ start }} {{ stop }} {{ num }} {{ time }}")
 
 _SCANS_WITHOUT_MOTORS = ['ct']
 _SCANS_WITH_MOTORS = ['ascan', 'dscan']
 _SPEC_SCAN_NAMES = _SCANS_WITH_MOTORS + _SCANS_WITHOUT_MOTORS
 _NOT_IMPLEMENTED_SCAN = 'Other'
-_BLUESKY_PLAN_NAMES = ['Scan', 'RelativeScan', 'Count']
+_BLUESKY_PLAN_NAMES = ['scan', 'relative_scan', 'count']
```
Comment (tacaswell) [inline suitcase/spec.py]: Are these the names you want?  We can make them have `['ascan', 'dscan', 'ct']` which will simplify a bunch of this.

Comment (ericdill) [inline suitcase/spec.py]: If you're ok with doing that, I think it makes a lot of sense to alias these things to whatever their spec names are

Comment (ericdill) [inline suitcase/spec.py]: @tacaswell is there a bluesky PR that you can link here so I know when these are renamed to ['ascan', 'dscan', 'ct']?

Comment (tacaswell) [inline suitcase/spec.py]: no because I have not done it yet.

Comment (ericdill) [inline suitcase/spec.py]: Hah fair enough.  I'll let this sit until that time then

Comment (tacaswell) [inline suitcase/spec.py]: for those following at home relevant pr is https://github.com/NSLS-II/bluesky/pull/457

**Comment by danielballan** [comment]
NSLS-II/bluesky#457 is in. Is this ready to merge?

**Comment by ericdill** [comment]
I have updated the names to what is in nsls-ii/bluesky#457, but I need to give this another close read to make sure this code respects the various keys that bluesky is inserting into the RunStart documents

**Comment by ericdill** [comment]
superseded by #16

**Pull request closed without merging**